src/components/htmlMediaHelper.js

```diff
@@ -30,30 +30,11 @@ define(['appSettings', 'browser', 'events'], function (appSettings, browser, eve
         return false;
     }
 
-    function enableHlsShakaPlayer(item, mediaSource, mediaType) {
+    function enableHlsShakaPlayer() {
         /* eslint-disable-next-line compat/compat */
         if (!!window.MediaSource && !!MediaSource.isTypeSupported) {
```

Comment:
`window.MediaSource` is `undefined` on WebOS 1.2 emulator. _No Shaka for you._

Reply:
It'll fall back down to the native player :)

HLS.js acts very much the same and doesn't work without MSE (Per the readme: `It relies on HTML5 video and MediaSource Extensions for playback.`)